hub/admin/extend_user.py

```diff
@@ -66,6 +67,30 @@ class UserCreationForm(DjangoUserCreationForm):
     )
 
 
+class OrgInline(admin.StackedInline):
+    model = OrganizationUser
+    verbose_name_plural = 'Organization'
+    view_on_site = False
+    list_display = [
+        'user',
+        'organization',
+        'is_admin',
+    ]
+    raw_id_fields = ('user', 'organization')
+    readonly_fields = ('active_subscription_status',)
```

Comment:
The 'Active Subscription' field shows up even when Stripe is disabled. What if you hid it then, since nobody has a subscription when Stripe is disabled?

```suggestion
    readonly_fields = settings.STRIPE_ENABLED and ('active_subscription_status',) or []
```

(I didn't explain the difference between servers where Stripe is enabled and disabled, so this one's on me. 😁)